ui/src/app/dashboard/grouped-summary/grouped-summary.component.css

```diff
@@ -1,14 +1,92 @@
-.mat-card {
+/*TODO(zachx): fixing the problem that expanding  a card reshuffles other cards */
+#container {
+  display: inline-block;
+  width: 25rem;
   margin: 1rem;
+
+}
+
+mat-card {
+  min-height: 220px;
+}
+
+div.expanded mat-card {
+  padding-bottom: 0;
+  /*min-height should plus removed padding-bottom, which is 24px by default*/
+  min-height: 244px;
+}
+
+div.collapsed mat-card {
+  padding-bottom: 0;
+  /*min-height should plus removed padding-bottom, which is 24px by default*/
+  min-height: 244px;
 }
 
-table a {
+div.collapsed clr-icon.arrow {
+  transform: rotate(180deg);
+}
+
+div.expanded clr-icon.arrow {
+  transform: rotate(0deg);
+}
+
+a {
   text-decoration: none;
   color: #000000;
+  margin-right: 40%
+}
+
+td.count a {
+  color: #555555;
+}
+
+a.zero {
+  color: #CCCCCC !important;
 }
 
-.mat-card-title {
-  font-size: 1.2rem;
-  color: gray
+tr > th.label {
+  width: 30%;
+  height: 2rem;
+  font-size: large;
 }
 
+table {
+  width: 100%;
+}
+tr > td.label-value {
+  width: 30%;
+  height: 1.3rem;
+}
+
+tr > td.count {
```

Comment:
Ideally the top-left cell would have text-align: left. Is this possible? This only matters for line-wrapped headers, but for example, "Job Name" looks wrong.

Reply:
Wrapping text with a span, fixed.